src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -1312,23 +1317,31 @@ class spell_pri_prayer_of_mending_SpellScriptBase : public SpellScript
     SpellEffectInfo const* _healEffectDummy;
 };
 
-// 33076 - Prayer of Mending
-class spell_pri_prayer_of_mending : public spell_pri_prayer_of_mending_SpellScriptBase
+// 33076 - Prayer of Mending (Dummy)
+// 411097 - Priest Holy 10.1 Class Set 2pc (Chooser)
+class spell_pri_prayer_of_mending_dummy : public spell_pri_prayer_of_mending_SpellScriptBase
 {
-    PrepareSpellScript(spell_pri_prayer_of_mending);
+    PrepareSpellScript(spell_pri_prayer_of_mending_dummy);
+
+    void FilterTargets(std::list<WorldObject*>& targets)
+    {
+        Trinity::SelectRandomInjuredTargets(targets, 1, true);
+    }
 
     void HandleEffectDummy(SpellEffIndex /*effIndex*/)
     {
-        CastPrayerOfMendingAura(GetCaster(), GetHitUnit(), GetEffectValue());
+        CastPrayerOfMendingAura(GetCaster(), GetHitUnit(), GetEffectValue(), true);
```

Comment:
Not sure if this GetEffectValue() would be correct for spell 411097. That one is BasePoints 0, whereas 33076 is 4. In any case, with no modifiers from auras, why are initial stacks at 5 in-game if GetEffectValue() is sending 4?